src/components/Calculator.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable linebreak-style */
+function Calculator() {
```

Comment:
- [ ] The `Calculator.js` component should be class-bassed (In the future for this project you'll see why this is important). Please fix this important issue. 🐶